backend/docs/developers/setup-local-env.rst

```diff
@@ -43,19 +44,36 @@ You need the following libraries and/or programs:
        npm run build
 
 #. Run the migrations with ``src/manage.py migrate``
+
+.. note::
+
+       If you get an error about the ``django.contrib.gis`` module,
+       you need to install the GDAL library and/or enable the extension or give superuser privileges. See the requirements in the Dockerfile.
```

Comment:
You need indeed to have Gdal (in the docker file it's done with `apt install gdal-bin`) and install the extension in the database (see previous comment).